### ISSUE TITLE
 No resource or method named `action_start'

### ISSUE THREAD

**Issue by dcparker88** [hw-cookbooks/runit#131]
Hello,

Going between 1.6.0 and 1.7.0 is giving me an error. I can't seem to find a change that would cause this in the commits. The error is:

```
 NoMethodError
           -------------
           No resource or method named `action_start' for `Chef::Provider::RunitService ""'
```

my runit_service resource is configured:

```
runit_service 'statsd' do
  action [:enable, :start]
  default_logger true
  options ({
    :user => node['statsd']['username'],
    :statsd_dir => node['statsd']['dir'],
    :conf_dir => node['statsd']['conf_dir'],
    :nodejs_bin => node['nodejs']['path']
  })
```

It looks like it's not liking the start action, but I can't figure out why. It works fine with 1.6.

Any help is appreciated, thanks!

**Comment by chrisroberts** [comment]
What version of chef are you running?

**Comment by dcparker88** [comment]
`Chef: 11.18.12`

I was able to reproduce in `Chef: 11.16.4` too

**Comment by chrisroberts** [comment]
Indeed. Looks like the `:start` action wasn't provided within the provider. I'm getting a fix implemented + coverage in the testing and should have a PR up shortly.

**Comment by dcparker88** [comment]
Yeah, I saw it wasn't provided but couldn't find a commit to show where it was removed. Thanks though :+1:

**Comment by cwjohnston** [comment]
Resolved by #133, which has been released in v1.7.2

**Comment by dcparker88** [comment]
thanks! 1.7.2 converged successfully :+1:
